src/components/Markdown/styled.elements.tsx

```diff
@@ -141,7 +141,7 @@ export const StyledMarkdownBlock = styled(
   }
 
   table tr {
-    background-color: #fff;
+    background-color: ${({ theme }) => theme.background};
```

Comment:
do we need this change? seems like it is handled in the change for theme. where is this getting applied?

Reply:
Ah, this is actually how the theme is applied throughout redoc! It's not anything magical - just an object that's referenced everywhere like this.